tests/integration/test_github/test_configuration/test_invalid.py

```diff
@@ -0,0 +1,247 @@
+                                              'sched_scan_validate_parameters(): ERROR').result()
+    elif tags_to_apply == 'invalid_interval':
+        wazuh_log_monitor.start(timeout=global_parameters.default_timeout,
+                                callback=callback_detect_interval_err,
+                                accum_results=1,
+                                error_message='Did not receive expected '
+                                              'sched_scan_validate_parameters(): ERROR').result()
+    elif tags_to_apply == 'invalid_curl_max_size':
+        wazuh_log_monitor.start(timeout=global_parameters.default_timeout,
+                                callback=callback_detect_curl_max_size_err,
+                                accum_results=1,
+                                error_message='Did not receive expected '
+                                              'sched_scan_validate_parameters(): ERROR').result()
+    elif tags_to_apply == 'invalid_time_delay':
+        wazuh_log_monitor.start(timeout=global_parameters.default_timeout,
+                                callback=callback_detect_time_delay_err,
+                                accum_results=1,
+                                error_message='Did not receive expected '
+                                              'sched_scan_validate_parameters(): ERROR').result()
+    elif tags_to_apply == 'invalid_org_name':
+        wazuh_log_monitor.start(timeout=global_parameters.default_timeout,
+                                callback=callback_detect_org_name_err,
+                                accum_results=1,
+                                error_message='Did not receive expected '
+                                              'sched_scan_validate_parameters(): ERROR').result()
+    elif tags_to_apply == 'invalid_api_token':
+        wazuh_log_monitor.start(timeout=global_parameters.default_timeout,
+                                callback=callback_detect_api_token_err,
+                                accum_results=1,
+                                error_message='Did not receive expected '
+                                              'sched_scan_validate_parameters(): ERROR').result()
+    elif tags_to_apply == 'invalid_event_type':
+        wazuh_log_monitor.start(timeout=global_parameters.default_timeout,
+                                callback=callback_detect_event_type_err,
+                                accum_results=1,
+                                error_message='Did not receive expected '
+                                              'sched_scan_validate_parameters(): ERROR').result()
+    else:
+        wazuh_log_monitor.start(timeout=global_parameters.default_timeout,
+                                callback=callback_detect_read_err,
+                                accum_results=1,
+                                error_message='Did not receive expected '
+                                              'wm_gcp_read(): ERROR:').result()
```

Comment:
This could probably be merged using a map.

Reply:
Done!